### PR TITLE
restore Ava-XS runner and disable tests that broke while it was off

### LINKED ISSUE #5498: xs tests passing blindly
## What is the Problem Being Solved?

The `test:xs-unit` command always passes, even when tests fail. For example in [yarn test (zoe)](https://github.com/Agoric/agoric-sdk/runs/6676047549?check_suite_focus=true):
```
$ ava-xs --verbose
# test script: test/unitTests/zoe/test-instanceAdminStorage.js
{
  status: 'not ok',
  message: 'running test script: Already locked down at TypeError: Prior lockdown (SES_ALREADY_LOCKED_DOWN) (SES_ALREADY_LOCKED_DOWN)',
  filename: 'test/unitTests/zoe/test-instanceAdminStorage.js',
  label: ''
}
# test script: test/unitTests/zoe/test-makeInvitation.js
{
  status: 'not ok',
  message: 'running test script: Already locked down at TypeError: Prior lockdown (SES_ALREADY_LOCKED_DOWN) (SES_ALREADY_LOCKED_DOWN)',
  filename: 'test/unitTests/zoe/test-makeInvitation.js',
  label: ''
}
0 tests passed
Done in 113.98s.
```

Notice that `ava-xs` is not detecting any tests running, so even though 0 passed the suite still passes.

## Description of the Design

First, make `ava-xs` count the failure of a test module to load as a failure. https://github.com/Agoric/agoric-sdk/pull/5398/commits/a29034acc5fc51250db285ece2ba1b1908f3dd36 might accomplish that.

Second, solve the [SES_ALREADY_LOCKED_DOWN](https://github.com/endojs/endo/blob/master/packages/ses/error-codes/SES_ALREADY_LOCKED_DOWN.md) error. Might be due library importing @endo/init or the swingset tools thing. Only the top-level test-*.js files should be doing those only-once imports.

~Third, fix any errors that have crept in while this test runner wasn't working.~ Too much to fix to accomplish this. Should have another ticket.

## Security Considerations
--
## Test Plan

♲

### PR DESCRIPTION
**Pull request by turadg**
closes: #5498

## Description

avaXS had stopped working because of a SES_ALREADY_LOCKED_DOWN issue that got introduced at some point. We didn't know it because the runner had a bug whereby a module failing to load would not be treated as a failure.

1. solve the runner bug with better error handling
2. solve the lockdown issue by handling new XS interop cases that have developed in the test code
3. skip testing on the remaining cases that haven't been solved (deferred to https://github.com/Agoric/agoric-sdk/issues/5500 ) 

### Security Considerations

none.

### Documentation Considerations

Indicate why the tests are disabled. Done.

### Testing Considerations

Disable tests that don't run.

Reviewers, do we also need a test of avaXS itself to check that this error would have been caught? It would definitely help but it would delay other work and the tests it runs are also already run in regular Ava.

### CODE REVIEW

File: packages/zoe/package.json [hunk 1]
```diff
@@ -73,6 +73,40 @@
   ],
   "ava-xs": {
     "exclude": [
+      "# path/url/import.meta stuff",
```
Comment (dckc) [inline packages/zoe/package.json:76]: cite #5500 in this comment? add TODO marker?

**Review by dckc — COMMENTED** [review]
I approve inasmuch as this matches the approach we agreed on.

I think it's cost-effective to get a more objective review, so for now at least, I defer to @kriskowal to approve in the github sense.

File: packages/ERTP/package.json [hunk 2]
```diff
@@ -57,7 +57,13 @@
     "exported.js"
   ],
   "ava-xs": {
-    "exclude": "swingsetTests"
+    "exclude": [
+      "swingsetTests",
+      "# fast-check unsupported",
```
Comment (turadg) [inline packages/ERTP/package.json:62]: do we need to test these in XS?

If not, maybe we have a different directory for tests that run under both Ava and XS?

Comment (dckc) [reply packages/ERTP/package.json:62]: It's not clear to me why fast-check is not supported. What am I missing? What were the symptoms when we tried?

Comment (turadg) [reply packages/ERTP/package.json:62]: They don't _all_ fail but it looked like something with fast-check,
```
. test/unitTests/test-amountProperties.js store
. test/unitTests/test-amountProperties.js reject promise keys
. test/unitTests/test-amountProperties.js passability of stores
. test/unitTests/test-amountProperties.js passability of store iters
. test/unitTests/test-amountProperties.js iteration fails with concurrent addition
. test/unitTests/test-amountProperties.js iteration succeeds with concurrent deletion
. test/unitTests/test-amountProperties.js provide for mapStores
FAIL (todo route console) (TypeError#1)
TypeError#1: ?: no function

{
  status: 'not ok',
  id: 160,
  message: 'isEqual is a (total) equivalence relation threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
F test/unitTests/test-amountProperties.js isEqual is a (total) equivalence relation
FAIL (todo route console) (TypeError#2)
TypeError#2: ?: no function

{
  status: 'not ok',
  id: 161,
  message: 'isGTE is a partial order with empty as minimum threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
F test/unitTests/test-amountProperties.js isGTE is a partial order with empty as minimum
FAIL (todo route console) (TypeError#3)
TypeError#3: ?: no function

{
  status: 'not ok',
  id: 162,
  message: 'add: closed, commutative, associative, monotonic, with empty identity threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
F test/unitTests/test-amountProperties.js add: closed, commutative, associative, monotonic, with empty identity
FAIL (todo route console) (TypeError#4)
TypeError#4: ?: no function

{
  status: 'not ok',
  id: 163,
  message: 'subtract: (x + y) - y = x; (y - x) + x = y if y >= x threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
F test/unitTests/test-amountProperties.js subtract: (x + y) - y = x; (y - x) + x = y if y >= x
FAIL (todo route console) (TypeError#5)
TypeError#5: ?: no function

{
  status: 'not ok',
  id: 164,
  message: 'minmax threw: ?: no function',
  filename: 'test/unitTests/test-amountProperties.js',
  label: ''
}
```

Comment (dckc) [reply packages/ERTP/package.json:62]: ok... fast-check is unsupported in that tests that don't use it don't seem to work and we're not investing in whatever it takes to get them to work yet. fair enough.

for future reference, now that `getStackString()` is working, that's where I would pick up on "todo route console".

**Review by kriskowal — APPROVED** [review]
Very much in favor of gradually improving ava-xs as needed. We don’t have a ground to expect equivalence even with a lot of effort.

We will Real Soon Now™ be able to subsume some of the idiosyncrasies into fully realized Endo support for CommonJS and `import.meta`. The result will share a lot of the same structure, but we might be able to increase parity a little.

**Pull request merged**